go/ct/driver/run.go

```diff
@@ -97,6 +97,14 @@ func doRun(context *cli.Context) error {
 	}
 
 	opRun := func(state *st.State) rlz.ConsumerResult {
+		result := rlz.ConsumeContinue
+		defer func() {
+			if r := recover(); r != nil {
+				result = rlz.ConsumeAbort
+				issuesCollector.AddIssue(state, fmt.Errorf("failed to process input state %v: %w", state, err))
```

Comment:
maybe change the error msg to differentiate it from the usual "failures" to something that reflects it faild due to a panic or is `err` the msg from the panic ?

Reply:
I will change the string, to emphasize that the execution panicked.